packages/mobile/package.json

```diff
@@ -17,7 +17,7 @@
     "dev:clean-android": "cd android && ./gradlew clean",
     "dev:send-text": "adb shell input text",
     "dev:reload": "adb shell input text \"RR\"",
-    "dev:emulator": "bash ./scripts/start_emulator.sh &",
+    "dev:emulator": "$ANDROID_HOME/emulator/emulator -avd Nexus_5X_API_28_x86 -no-boot-anim",
```

Comment:
Funny, I also fixed this in my PR here, adding a list and a run command:
https://github.com/celo-org/celo-monorepo/pull/3765/files#diff-4bbd284d4cdf8b13bb5dc9aed93f552bR20

Reply:
Awesome